app/controllers/api/v1/short_form_controller.rb

```diff
@@ -6,159 +6,31 @@ class Api::V1::ShortFormController < ApiController
 
   def submit
     logger.debug "application_api_params: #{application_api_params}"
-    short_form_validator = ShortFormValidator.new(application_api_params)
-
-    if short_form_validator.valid?
-      application = custom_api_application_service.submit(application_api_params)
-      logger.debug "application submit response: #{application}"
-      # if submitting a supplemental application we will re-fetch updated data from salesforce
-      if params[:supplemental]
-        application = soql_application_service.application(application[:id] || application_api_params[:id], { include_lease: false })
-        logger.debug "updated application: #{application}"
-      end
-
-      render json: { application: application }
-    else
-      render status: 422, json: { errors: short_form_validator.errors.full_messages || 'Unknown Error' }
+    # short_form_validator = ShortFormValidator.new(application_api_params)
+    # if short_form_validator.valid?
+    # FIXME: validatate / permit on the params
+    # Unpermitted parameter: :primary_language
```

Comment:
We can remove this too, and probably remove the whole short_form_validator file. under apps/services/short_form_validator.rb
```suggestion
    # TODO: Consider adding a validator to check that all required params are present
```